### PR TITLE
Landing Page with user proposals

### PR DESCRIPTION
**Pull request by tiagoalvesdulce**
Created a new route called `user/:userid/proposals` that will be the new landing page where the user is redirected to after the login. I am not sure if we want the `userid` on the URL, so LMK your thoughts on this.

Closes https://github.com/decred/politeiagui/issues/163.

### CODE REVIEW

**Comment by sndurkin** [comment]
I was thinking that the proposal list could be on the main user page, so the url would be `/user/:userid`, but I'm not sure what we would do with the Reset Password or the identity management area.

@go1dfish what do you think?

**Comment by tiagoalvesdulce** [comment]
Yes, I think the headers are a good idea. Also, I'll be adding a Dropdown (https://github.com/decred/politeiagui/issues/193) on my next PR that will add a way to easily access `user/proposals` from the UI.

File: src/connectors/proposals.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ export default connect(
     isAdmin: sel.isAdmin,
     proposals: sel.vettedProposals,
     error: sel.vettedProposalsError,
-    isLoading: sel.vettedProposalsIsRequesting
+    isLoading: sel.vettedProposalsIsRequesting,
+    header: () => "Vetted Proposals"
```
Comment (sndurkin) [inline src/connectors/proposals.js]: Let's call this "Active Proposals" for now.

File: src/components/snew/Content.js [hunk 2]
```diff
@@ -11,8 +11,10 @@ export const CustomContent = ({
   proposals,
   isLoading,
   error,
+  header,
   ...props
 }) => {
+  console.log(header);
```
Comment (sndurkin) [inline src/components/snew/Content.js]: I think you forgot to remove this.

**Pull request merged**